lib/v2/fisher-spb/news.js

```diff
@@ -16,7 +16,7 @@ module.exports = async (ctx) => {
         responseType: 'buffer',
     });
 
-    const $ = cheerio.load(iconv.decode(response.data, 'windows-1251'));
+    const $ = cheerio.load(iconv.decode(response.data, 'utf-8'));
```

Comment:
Cheerio can handle utf-8 without the need of iconv. Please remove as well.

Reply:
done